tools/Microsoft.TemplateEngine.Authoring.CLI/Commands/validate/ValidateCommand.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.CommandLine;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.TemplateEngine.Abstractions;
+using Microsoft.TemplateEngine.Edge;
+using Microsoft.TemplateEngine.Edge.Settings;
+
+namespace Microsoft.TemplateEngine.Authoring.CLI.Commands
+{
+    internal class ValidateCommand : ExecutableCommand<ValidateCommandArgs>
+    {
+        private const string CommandName = "validate";
+
+        private readonly Argument<string> _templateLocationArg = new("template-location")
+        {
+            Description = "The location of template(s) to validate",
+            // 0 for case where only path is specified
```

Comment:
This comment is confusing me :-) Always a single path argument is required - correct?

Reply:
Honestly, I don't remember and don't understand this comment either, so removed it.